lib/child_process.js

```diff
@@ -114,6 +109,82 @@ exports.exec = function(command /*, options, callback*/) {
                           opts.callback);
 };
 
+function normalizeArgsOptions(allArgs, defaultOption) {
+  var pos = 1, arrayPos = -1, objectPos = -1, args;
```

Comment:
I think we're using `let` and `const` for new code these days, best switch where appropriate